src/Entity/Entity.php

```diff
@@ -380,6 +381,7 @@ public function getClaims() {
 
 	/**
 	 * @since 0.7.3
+	 * @deprecated since 3.1, use FingerprintProvider instead
```

Comment:
This does not make much sense. This method implements the interface, how can one use the interface instead of it's implementation?